src/auth/strategies/magic-link.strategy.ts

```diff
@@ -47,7 +47,7 @@ export class MagicLinkStrategy extends PassportStrategy(
         user,
       } as MagicLinkContext;
     } catch {
-      return this.fail('no_account_found');
+      return this.fail('Error: No Iron Fish account exists for this email.');
```

Comment:
I think it'd be nice for consistency to return objects here with a `code` and `message`, like we do [here](https://github.com/iron-fish/ironfish-api/blob/master/src/users/users-updater.ts#L30-L45). This could look like
```suggestion
      return this.fail({
        code: 'user_not_found',
        message: 'User not found for this email',
      });
```

What do you think?